cdc/entry/mounter_group.go

```diff
@@ -0,0 +1,131 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package entry
+
+import (
+	"context"
+	"strconv"
+	"sync/atomic"
+	"time"
+
+	"github.com/pingcap/errors"
+	"github.com/pingcap/tiflow/cdc/model"
+	"github.com/pingcap/tiflow/pkg/filter"
+	"golang.org/x/sync/errgroup"
+)
+
+// MounterGroup is a group of mounter workers
+type MounterGroup interface {
+	Run(ctx context.Context) error
+	AddEvent(ctx context.Context, event *model.PolymorphicEvent) error
+}
+
+type mounterGroup struct {
+	schemaStorage  SchemaStorage
+	inputCh        []chan *model.PolymorphicEvent
+	tz             *time.Location
+	filter         filter.Filter
+	enableOldValue bool
+
+	workerNum int
+	index     uint64
+
+	changefeedID model.ChangeFeedID
+}
+
+const (
+	defaultMounterWorkerNum = 16
+	defaultOutputChanSize   = 64
```

Comment:
Is this a typo? Should it be `defaultInputChanSize`?

Reply:
Is 64 enough for a large workload in which `mounter.DecodeEvent()` will spend more time?